### ISSUE TITLE
Picker not working with BufferGeometry

### ISSUE THREAD

**Issue by aothms** [jovyan/pythreejs#152]
Thanks for your awesome work.

Consider the code below, roughly taken from the voxel painter demo [1]. The example works picking a `BoxGeometry`, but not picking a `BufferGeometry`.

Is this a limitation in ThreeJS? I couldn't find much about what this `Picker` class actually is, it is somehow related to the `Raycaster` [1] (which I thought ought to work on `BufferGeometry`)?

ThreeJS does not have a selection function based on the GPU (i.e render a 1px buffer with unique object colors and read pixel to determine object under mouse cursor)?

[1] https://threejs.org/docs/#api/core/Raycaster

~~~py
from pythreejs import *
from IPython.display import display
from ipywidgets import HTML

import numpy

cube_geo = BoxGeometry(
    width = 2,
    height = 2,
    depth = 2
)

# comment ...

np_vertices = numpy.array([
    (0,0,0),
    (2,0,0),
    (0,2,0),
    (2,2,0),
], dtype=numpy.float32)

np_faces = numpy.array([
    0,1,2, 1,3,2
], dtype=numpy.uint32)

cube_geo = BufferGeometry(attributes={
    'position': BufferAttribute(np_vertices),
    'index'   : BufferAttribute(np_faces)
})

# ... till here to have working code

cube_mat = MeshLambertMaterial(
    color = '#feb74c',
    shading = 'FlatShading'
)

cube = Mesh(
    geometry = cube_geo,
    material = cube_mat
)

html = HTML("not clicked")

def f(*args):
    html.value = "clicked!"

pickable_objects = Group()
pickable_objects.add(cube)

click_picker = Picker(
    controlling = pickable_objects,
    event = 'dblclick'
)
click_picker.observe(f, names=["point"])

camera = PerspectiveCamera(
    position = [10,10,10],
    lookAt = [1,1,1]
)

scene = Scene(
    children = [
        pickable_objects,
        AmbientLight(
            color = '#606060'
        )
    ]
)

renderer = Renderer(
    camera = camera,
    scene = scene,
    controls = [
        OrbitControls(
            controlling = camera
        ),
        click_picker
    ]
)

display(html, renderer)
~~~

[1] https://github.com/jovyan/pythreejs/blob/master/examples/threejs/voxel_painter.ipynb

**Comment by aothms** [comment]
My guess is that this is not so much due to the BufferGeometry, but due to the lack of UV coords and an issue with the code, see #153

**Comment by vidartf** [comment]
Is this fixed with the merge of #153?

**Comment by aothms** [comment]
Yes, thanks for merging!
